TRACEABILITY.md

```diff
@@ -0,0 +1,43 @@
+# Traceability utility
+
+This utility provides several methods to facilitate traceability of specs against test cases. 
+
+## Trace of specifications
+
+This utility has two working modes:
+ - Add specs at either "it" (test) level
+ - Add specs at "describe" (suite) level
+
+### Tracing specs at "it" (test) level 
+Setting specs at "it" level, the specs related to each "it" test are added with the `addCoveredSpecs()` method:
+```typescript
+TraceabilityUtility.addCoveredSpecs(['SwS6015', 'SwS6016']);
+it('Perform a Log in', async () => {
+    await LoginActionService.login(true, UserCredentialsList.getDefault());
+    ...
+});
+```    
+Then at suite level all specs added at "it" level can be dumped to the allure report at once with the `dumpCoveredSpecs()` method:
+```typescript
+describe('WDIO-TC-AT-SPECIFIC_TEST', () => {
+    beforeEach(async () => {
```

Comment:
I just have a doubt: is it mandatory to perform this in the beforeEach block, or could it be performed in the beforeAll block (just once for the whole run of the describe)?
Then, if that detail was relevant, it could be added to this info